### PR TITLE
Tj edits on HEASoft, VOEvent, misc

### PR DESCRIPTION
**Pull request by trjaffe**
Adding some references to long-standing multimission software and a bit about VOEvent and the GCN.  Minor mods elsewhere.  

This will start as a draft PR so we can discuss as necessary and revise before merging.  (Note also my laptop was recently updated and latex broken, so I haven't compiled it yet.)

### CODE REVIEW

File: VOHE-Note.tex [hunk 1]
```diff
@@ -1006,9 +1007,12 @@ \subsection{Use of Datalink for HE products}
...
-dereferencable URLs or alternbatively to different fragments of the same drefereencable URL as stated by the DataLink specification.
+In the second option, we would provide links to the relevant information from the base "event-list" (\ref{sec:event-bundlle-or-list}) or product exposed in the discovery service. This could be done using Datalink and a list of links to each contextual information such as the \gls{IRF}s. The Event-list Context Data Model (see \ref{sec:EventListContext}) would provide the concepts and vocabulary to characterise the \gls{IRF}s and other information relevant to the analysis of an event-list. These specific concepts and terms describing the various flavors of \gls{IRF}s and \gls{GTI}s will be given in the semantics and content\_qualifier FIELDS of the DataLink response to qualify the links. The different links can point to different
+dereferencable URLs or alternatively to different fragments of the same dereferencable URL as stated by the DataLink specification.
 
+\subsection{VOEvent modernization}\lable{sec:voevent_he}
```
Comment (mservillat) [inline VOHE-Note.tex]: \lable --> \label

File: ivoatex
Comment (mservillat) [inline ivoatex:1]: This should not be part of the pull request, could it be removed ?

Comment (trjaffe) [reply ivoatex:1]: Should be fixed.

File: VOHE-Note.tex (hunk 1 above)
Comment (trjaffe) [reply VOHE-Note.tex]: Should be fixed.
